docs/tutorials/constraints.rst

```diff
@@ -339,11 +310,15 @@ The resulting list of booleans indicates that the only target separated by
 5 and 30 degrees from Vega is Albireo. Following this pattern, you can design
 arbitrarily complex criteria for constraints.
 
-To see the (target x time) array for the non-boolean score::
+By default, calling a constraint will try to broadcast the time and target arrays
+against each other, and raise a `ValueError` if this is not possible. To see the
+(target x time) array for the constraint, there is an optional "grid_times_targets"
```

Comment:
Small correction – would you please make `"grid_times_targets"` -> ``` `grid_times_targets` ```? That way it will be rendered nicely in the docs.